google-cloud-trace/lib/google/cloud/trace/service.rb

```diff
@@ -160,12 +137,6 @@ def execute
           # GaxError wraps BadStatus, but exposes it as #cause
           raise Google::Cloud::Error.from_error(e.cause)
         end
-
-        def load_grpc
-          require "grpc"
-          require "google/cloud/trace/patches/active_call_with_trace"
```

Comment:
These patches still need to be required somewhere, maybe just in google/cloud/trace.rb. They just need to be loaded after grpc.

Reply:
Would it work to add these requires to `lowlevel_client` right before the `TraceServiceClient` object was created?

Reply:
@dazuma I restored the requires in the `lowlevel_client` method. Can you revisit this and make sure that this is correct?

Reply:
This should be okay. Thanks @blowmage !